tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -256,6 +256,10 @@ TEST_P(BackendCorrectnessTest, dataParallelStackingTest) {
     bb.createDeallocActivationInst("dealloc", act);
   }
 
+#ifndef GLOW_WITH_CPU
```

Comment:
this is fine due to usage of `MockCPUBackend`. Would be good to get rid of this as well.